shared/common-adapters/icon.desktop.tsx

```diff
@@ -110,6 +108,8 @@ class Icon extends Component<Props, void> {
           hoverColor: 'inherit',
         }
       } else {
+        // invert the colors here so it reflects the colors in current theme
+        const invertedColors = invert(Styles.globalColors)
```

Comment:
how expensive an operation is `invert`? If a page has hundreds of icons on it on mobile will this be a problem? At the very least maybe we should invert only the colors used here?

Reply:
it takes an object K -> V and returns another object with V -> K, so I can't imagine it's super cheap/performant. There should almost certainly be a better solution than this, but I'm not sure what that would look like.

Reply:
It's fine